libs/evm2wasm/CMakeLists.txt

```diff
@@ -7,5 +7,9 @@ add_library(libevm2wasm STATIC
 )
 set_target_properties(libevm2wasm PROPERTIES OUTPUT_NAME evm2wasm)
 
+if(BUILD_SHARED_LIBS)
+  set_target_properties(libevm2wasm PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
```

Comment:
You probable could always enable it.

Reply:
Could or should?

Reply:
It should not affect the performance so the configuration will be simpler (and less number of combinations). But this is not required.

Reply:
It does affect the performance, but not sure if that is any significant nowadays compared to the overall complexity of the code anyway.

Reply:
It will be a question when we try compiling this to wasm, but that is a very long way ahead :)

Reply:
Many modern linux distributions build with PIC by default.